cloudapi/config_test.go

```diff
@@ -48,6 +48,7 @@ func TestConfigApply(t *testing.T) {
 		PushRefID:                       null.NewString("PushRefID", true),
 		WebAppURL:                       null.NewString("foo", true),
 		NoCompress:                      null.NewBool(true, true),
+		StopOnError:                     null.NewBool(true, true),
```

Comment:
I think this should be `false` by default. First, because it would be a minor breaking change. 

Second, because in most situations, people wouldn't run `k6 run --out cloud` with `--no-summary` or `--no-thresholds`, so even if the cloud starts rejecting their metrics mid-test, they'd still get some value out of the results by examining the local summary output.

Reply:
This is just the test

Reply:
:man_facepalming: sorry :sweat_smile: